serverless/lambda/src/test/java/uk/gov/di/lambdas/CheckUserExistsHandlerTest.java

```diff
@@ -45,6 +46,7 @@ public void setup() {
         handler =
                 new CheckUserExistsHandler(
                         validationService, authenticationService, sessionService);
+        reset(authenticationService);
```

Comment:
Looks good, but what do we need the reset for?

Reply:
Was debugging the test and wasn't sure I was getting the correct response from this mock, so put this in to make sure it was reset.  Might be able to remove it, will give it a go.